archi@4.6.0: download via aria2 failed

```
scoop install archi
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Installing 'archi' (4.6.0) [64bit]
Starting download with aria2 ...
Download: [#2a9750 0B/0B CN:1 DL:0B]
Download: 03/11 20:23:05 [ERROR] CUID#8 - Download aborted. URI=https://www.archimatetool.com/downloads/51f64a6b21595fe0b/Archi-Win64-4.6.0.zip
Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://www.archimatetool.com/downloads/51f64a6b21595fe0b/Archi-Win64-4.6.0.zip
Download:   -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 2a9750|ERR |       0B/s|C:/Users/maverick/scoop/cache/archi#4.6.0#https_www.archimatetool.com_downloads_51f64a6b21595fe0b_Archi-Win64-4.6.0.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.
ERROR Download failed! (Error 3) Resource was not found
ERROR https://www.archimatetool.com/downloads/51f64a6b21595fe0b/Archi-Win64-4.6.0.zip
    referer=https://www.archimatetool.com/downloads/51f64a6b21595fe0b/
    dir=C:\Users\maverick\scoop\cache
    out=archi#4.6.0#https_www.archimatetool.com_downloads_51f64a6b21595fe0b_Archi-Win64-4.6.0.zip

ERROR & 'C:\Users\maverick\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\maverick\scoop\cache\archi.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=22996 --continue
```

--- a/bucket/archi.json
+++ b/bucket/archi.json
@@ -5,7 +5,7 @@
     "license": "MIT",
     "architecture": {
         "64bit": {
-            "url": "https://www.archimatetool.com/downloads/51f64a6b21595fe0b/Archi-Win64-4.6.0.zip",
+            "url": "https://www.archimatetool.com/downloads/4.6.0/Archi-Win64-4.6.0.zip",
             "hash": "md5:78f2f45b32386154bbf28bef512c9e78"
         }
     },

--- a/bucket/archi.json
+++ b/bucket/archi.json
@@ -20,5 +20,12 @@
     "checkver": {
         "url": "https://github.com/archimatetool/archi/releases",
         "regex": "release_([\\d.]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://www.archimatetool.com/downloads/$version/Archi-Win64-$version.zip"
+            }
+        }
     }
 }

--- a/bucket/archi.json
+++ b/bucket/archi.json
@@ -26,6 +26,10 @@
             "64bit": {
                 "url": "https://www.archimatetool.com/downloads/$version/Archi-Win64-$version.zip"
             }
+        },
+        "hash": {
+            "url": "$url.MD5",
+            "regex": "\\($basename\\)\\s+=\\s+$md5"
         }
     }
 }

